domain_info.go

```diff
@@ -0,0 +1,99 @@
+package epp
+
+import (
+	"bytes"
+	"encoding/xml"
+	"fmt"
+	"strings"
+)
+
+type DomainStatus struct {
```

Comment:
Please add Go doc comments to any new exported types

Reply:
I'm suprised I didn't get a complaint from `go fmt`.  I'll add those

Reply:
I’d be game to add a https://staticcheck.io linter to this repo’s GitHub Actions workflows.